sql/hive/src/test/scala/org/apache/spark/sql/hive/HiveDataFrameWindowSuite.scala

```diff
@@ -189,7 +189,7 @@ class HiveDataFrameWindowSuite extends QueryTest {
       df.select(
         $"key",
         last("value").over(
-          Window.partitionBy($"value").orderBy($"key").rangeBetween(1, Long.MaxValue))
+          Window.partitionBy($"value").orderBy($"key").rangeBetween(-1, 0))
```

Comment:
Yes, it was wrong. The default behavior when Hive parses the this partial frame definition '..RANGE 1 preceding...' is to expand this into '..RANGE BETWEEN 1 PRECEDING AND CURRENT ROW...', and not into '..RANGE BETWEEN 1 FOLLOWING AND UNBOUNDED PRECEDING...'. So the test was wrong.

But this still strikes me as odd, how did this pass previous tests? Did the default behavior change?